lib/DhallToCabal/ConfigTree.hs

```diff
@@ -113,8 +113,12 @@ subExpr f = \case
   App a b ->
     App <$> f a <*> f b
 
-  Let a b c d ->
-    Let a <$> traverse f b <*> f c <*> f d
+  Let as b ->
+    Let
+      <$> traverse
+            (\(Binding v n c) -> Binding v <$> traverse f n <*> f c)
```

Comment:
This function (`subExpr`) can be replaced wholesale with `subExpressions`, which is new in `dhall` 1.19.